packages/query/src/common/remote-config/remote-config.query.ts

```diff
@@ -129,7 +129,7 @@ export function useConfigTokensEnabledByDefault(): string[] {
 
 export function useConfigTokenTransferFeeEstimations() {
   const config = useRemoteConfig();
-  return get(config, 'tokenTransferFeeEstimations', []);
+  return get(config, 'tokenTransferFeeEstimations', undefined);
```

Comment:
This is the default argument that if undefined, returns undefined

Reply:
yeah, I'm working now in extension on fallback, done this pr before we decided to do that in thread

Reply:
actually can close this one prob